### PR TITLE
fix: Updating procedure for namespace provisioning in advance

### PR DESCRIPTION
**Pull request by ibuziuk**
## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/provisioning-namespaces-in-advance.adoc [hunk 1]
```diff
@@ -11,6 +11,15 @@ You can provision workspaces {orch-namespace}s in advance, rather than relying o
 
 .Procedure
 
+* Disable automatic namespace provisioning on the `CheCluster` level:
```
Comment (deerskindoll) [inline modules/administration-guide/pages/provisioning-namespaces-in-advance.adoc]:
```suggestion
. Disable automatic namespace provisioning on the `CheCluster` level:
```

File: modules/administration-guide/pages/provisioning-namespaces-in-advance.adoc [hunk 2]
```diff
@@ -11,6 +11,15 @@ You can provision workspaces {orch-namespace}s in advance, rather than relying o
...
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+devEnvironments:
+  defaultNamespace:
+    autoProvision: false
+----
+
 * Create the __<{orch-namespace}_name>__ {orch-namespace} for __<username>__ user with the following labels and annotations:
```
Comment (deerskindoll) [inline modules/administration-guide/pages/provisioning-namespaces-in-advance.adoc]:
```suggestion
. Create the __<{orch-namespace}_name>__ {orch-namespace} for __<username>__ user with the following labels and annotations:
```

**Review by deerskindoll — COMMENTED** [review]
lgtm

**Comment by ibuziuk** [comment]
@deerskindoll would be nice to backport for 3.16

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**